dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -58,50 +36,13 @@ static const uint8_t tracking_consent_granted = 1;
 static uint8_t tracking_consent = tracking_consent_pending; // 0 - PENDING, 1 - GRANTED, 2 - NOT-GRANTED
 
 
-std::string get_serialized_log(int signal,
-                               const char *signal_name,
-                               const char *error_message,
-                               const char *date,
-                               const std::string backtrace) {
-    std::string serializedLog = "{ ";
-    if (!rum_context.application_id.empty()) {
-        serializedLog.append(R"("application_id": ")").append(rum_context.application_id)
-                .append("\",");
-    }
-    if (!rum_context.session_id.empty()) {
-        serializedLog.append(R"("session_id": ")").append(rum_context.session_id)
-                .append("\",");
-    }
-    if (!rum_context.view_id.empty()) {
-        serializedLog.append(R"("view.id": ")").append(rum_context.view_id)
-                .append("\",");
-    }
-    // these values are either constants or they are marked as NonNull in JVM so we do not have
-    // to check them here.
-    char tags[105]; // max 105 characters for the environment name
-    snprintf(tags, sizeof(tags), "env:%s", main_context.environment.c_str());
-    serializedLog.append(R"("message": ")").append(main_context.generic_message).append("\",");
-    serializedLog.append(R"("service": ")").append(main_context.service_name).append("\",");
-    serializedLog.append(R"("logger.name": ")").append(main_context.logger_name).append("\",");
-    serializedLog.append(R"("ddtags": ")").append(tags).append("\",");
-    serializedLog.append(R"("status": ")").append(main_context.emergency_status).append("\",");
-    serializedLog.append(R"("date": ")").append(date).append("\",");
-    serializedLog.append(R"("error.stack": ")").append(backtrace).append("\",");
-    serializedLog.append(R"("error.message": ")").append(error_message).append("\",");
-    char formatted_signal_message[30];
-    const size_t messageSize =
-            sizeof(formatted_signal_message) / sizeof(formatted_signal_message[0]);
-    snprintf(formatted_signal_message,
-             messageSize, "%s: %d",
-             signal_name,
-             signal);
-    serializedLog.append(R"("error.signal": ")").append(formatted_signal_message).append("\",");
-    serializedLog.append(R"("error.kind": ")").append(main_context.error_kind).append("\"");
-    serializedLog.append(" }");
-    return serializedLog;
-}
+void crash_signal_intercepted(int signal,
```

Comment:
`signal` is a function, people usually prefer to use `signum` in such situations to avoid masking it